docs/cli.md

```diff
@@ -522,6 +523,26 @@ gcli tag update --tag tagA --rename newTag
 gcli tag update --tag tagA --comment "new comment"
 ```
 
+### Owners commands
+
+#### List an owner
+
+```bash
+gcli catalog details --name postgres --owner
+```
+
+#### Set an owner to a user
+
+```bash
+gcli catalog set --name postgres --user admin
+```
+
+#### Set an owner to a group
+
+```bash
+gcli catalog set --name postgres --group groupA
```

Comment:
Now you remove the "--owner" flag? from this sample, it's hard to understand it is setting the owner.

Reply:
As discussed in another thread, we can change the order of arguments in examples to make that clearer and/or add `--owner`.

Reply:
Currently, the code reflects what you wanted without an `--owner` flag.

Reply:
I would suggest we go with:
```
gcli catalog details --owner --metalake metalake_demo --name postgres
cli catalog set --owner  --user admin --name postgres
cli catalog set --owner --group groupA --name postgres
```
(Note the order of the options doesn't matter, but it's clearer to the user when ordered in this way.

Reply:
> I would suggest we go with:
> 
> ```
> gcli catalog details --owner --metalake metalake_demo --name postgres
> cli catalog set --owner --user admin --name postgres
> cli catalog set --owner --group groupA --name postgres
> ```
> 
> (Note the order of the options doesn't matter, but it's clearer to the user when ordered in this way.

As a compromise, I would suggest going this way, what do you think @shaofengshi @xunliu ?

Reply:
I've changed the code to use `--owner` flag for set so you can see the changes.